server/mergin/sync/private_api.yaml

```diff
@@ -26,13 +26,45 @@ paths:
           required: true
           schema:
             type: string
+        - name: page
+          in: query
+          description: page number
+          required: true
+          schema:
+            type: integer
+            minimum: 1
+            example: 1
+        - name: per_page
+          in: query
+          description: Number of results per page
+          required: true
+          schema:
+            type: integer
+            maximum: 50
+            example: 20
+        - name: order_params
+          in: query
+          description: Sorting fields e.g. expire DESC
```

Comment:
Can we unify pattern with get paginated projects (v1) ? "expire_desc" ??

Reply:
the one used now is newer (and already used at various places), it would be better to unify it with projects pagination the other way around - but because it is public api it is not that easy - what we can do is start using the new format of order params while keeping old functional and start updating clients